src/components/product-switcher/index.tsx

```diff
@@ -32,8 +32,8 @@ const getLastProduct = (products: Product[][]) => {
 }
 
 const ProductSwitcher: React.FC = () => {
-  const router = useRouter()
-  const currentProductSlug = router.asPath.split('/')[1] as ProductSlug
+  const currentPath = useCurrentPath()
```

Comment:
If I'm understanding your intention here, I think you also need to send two options in this `useCurrentPath` call:

```suggestion
  const currentPath = useCurrentPath({ excludeHash: true, excludeSearch: true })
```

Reply:
Perfect, thank you! 💯